### PR TITLE
tests: add missing check for setNewFlowCell

### PR DESCRIPTION
**Pull request by StephenHulme**
Discovered when investigating failing Vue updates (#1760)

#### Changes proposed in this pull request

- Add missing test for `setNewFlowCell` (cherrypicked from PR above)

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: tests/unit/components/ont/runs/ONTFlowcell.spec.js [hunk 1]
```diff
@@ -98,6 +98,11 @@ describe('ONTFlowcell', () => {
         ontFlowcell.setPoolTubeBarcode({ barcode: 'TRAC-1-B', position: 1 })
         expect(ontFlowcell.poolTubeBarcode).toEqual('TRAC-1-B')
       })
+
+      it('#setNewFlowCell', () => {
```
Comment (BenTopping) [inline tests/unit/components/ont/runs/ONTFlowcell.spec.js:102]: makes sense to add the test but currently failing CI

**Review by BenTopping — CHANGES_REQUESTED** [review]

File: tests/unit/components/ont/runs/ONTFlowcell.spec.js (hunk 1 above)
Comment (seenanair) [reply tests/unit/components/ont/runs/ONTFlowcell.spec.js:102]: Sorry, I didn't get it. Why are we adding this test?

Comment (seenanair) [reply tests/unit/components/ont/runs/ONTFlowcell.spec.js:102]: Oh, ok. Thank you for explaining the context. 👍

Comment (seenanair) [reply tests/unit/components/ont/runs/ONTFlowcell.spec.js:102]: Hmm, does it pass locally?

Comment (seenanair) [reply tests/unit/components/ont/runs/ONTFlowcell.spec.js:102]: I had a quick look. I think the position is a prop and component is mounted with a position 1. The setNewFlowCell is a store method which creates a new flowcell here at position 2., But, I suppose it won't change the position value of an already existing component.

**Pull request closed without merging**